src/utils/bugzilla/advancedSearchToRestApi.js

```diff
@@ -0,0 +1,13 @@
+const TRANSFORM_FIELD = {
+    chfieldfrom: 'creation_time',
+};
+
+const advancedSearchToRestApi = parameters => (
+    Object.keys(parameters).reduce((result, key) => {
+        const newKey = TRANSFORM_FIELD[key] ? TRANSFORM_FIELD[key] : key;
```

Comment:
I think you could achieve the same result by doing this instead `TRANSFORM_FIELD[key] || key'

Reply:
One day my brain will actually absorb this syntax. Thanks!